Sources/ExperienceEvent.swift

```diff
@@ -24,31 +24,59 @@ public class ExperienceEvent: NSObject {
     /// Optional free-form query data associated with this event
     @objc public var query: [String: Any]?
 
+    /// Adobe Experience Platform datastream identifier, used to override the default datastream identifier set in the Edge Configuration
+    @objc public private(set) var datastreamIdOverride: String?
+
+    /// Adobe Experience Platform datastream configuration, used to override the datastream configuration
```

Comment:
Nit doc (same for all occurrences):
Datastream identifier used to override the default datastream identifier set in the Edge configuration for this event
Datastream configuration used to override individual settings from the default datastream configuration for this event

Reply:
Can we use "Datastream configuration .. " for the property description too so that it does not get confused with a setting in Experience Platform